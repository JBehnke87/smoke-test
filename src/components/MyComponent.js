const MyComponent = (props) => {
    return (<div><h1>Hello!</h1></div>)
}

export default MyComponent;