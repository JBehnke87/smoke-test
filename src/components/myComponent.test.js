import React from 'react';
import MyComponent from './MyComponent';
import renderer from 'react-test-renderer';

test('renders the same way every time', () => {
  const tree = renderer.create( <MyComponent /> ).toJSON();
  expect(tree).toMatchSnapshot();
});