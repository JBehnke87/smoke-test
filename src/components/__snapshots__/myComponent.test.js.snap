// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the same way every time 1`] = `
<div>
  <h1>
    Hello!
  </h1>
</div>
`;
